src/hotspot/cpu/riscv/globals_riscv.hpp

```diff
@@ -116,7 +116,7 @@ define_pd_global(intx, InlineSmallCode,          1000);
   product(bool, UseZihintpause, false, EXPERIMENTAL,                             \
           "Use Zihintpause instructions")                                        \
   product(bool, UseZvbb, false, EXPERIMENTAL, "Use Zvbb instructions")           \
-  product(bool, UseZvfh, false, EXPERIMENTAL, "Use Zvfh instructions")           \
+  product(bool, UseZvfh, false, "Use Zvfh instructions")                         \
```

Comment:
Nit: Better to move it out of the `EXPERIMENTAL` group. Maybe immediately after the line for `UseZfh`.

Reply:
> Nit: Better to move it out of the `EXPERIMENTAL` group. Maybe immediately after the line for `UseZfh`.
> 

Currently the pattern is ordering by name, so I'd prefer to keep it. :)

> (Seems we should also auto-enable this feature through hwprobe like other non-experimental flags. I see macros `RISCV_HWPROBE_EXT_ZVFH` and `RISCV_HWPROBE_EXT_ZVFHMIN` in file os_cpu/linux_riscv/riscv_hwprobe.cpp, but neither of them are checked in function `RiscvHwprobe::add_features_from_query_result`)

Thanks, add test of RISCV_HWPROBE_EXT_ZVFH, but not RISCV_HWPROBE_EXT_ZVFHMIN, as we don't have the corresponding define and usage ext_Zvfhmin and so on.